File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1837.go
package main
 
import (
    "fmt"
)

func main() {
    var a, b, q, r int;

    fmt.Scanf("%d %d", &a, &b);

    q = a/b;
    r = a%b;
       
    if(r<0){
        if(q<0){ 
            q-=1;
            
        }else if(q>0){ 
            q+=1;
        }else{
            if(a-((q-1)*b)>=0){
                q-=1;
            } else{
                q+=1;
            }
        }
        r = a-(q*b);
    }
        
    fmt.Printf("%d %d\n", q, r);

}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1837.c
#include <stdio.h>

int main(){
    int a, b;
    scanf("%d %d", &a, &b);
    int q = a/b, r = a%b;
    
    if(r<0){
        if(q<0) q--;
        else if(q>0) q++;
        else{
            if(a-((q-1)*b)>=0){
                q--;
            } else{
                q++;
            }
        }
        r = a-(q*b);
    }
    
    printf("%d %d\n", q, r);
    
    return 0;
}
